modules/@apostrophecms/schema/ui/apos/components/AposInputAttachment.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <AposInputWrapper
+    :field="field" :error="error"
+    :uid="uid"
+  >
+    <template #body>
+      <div class="apos-attachment">
+        <label
+          class="apos-input-wrapper apos-attachment-dropzone"
+          :class="{ 'apos-attachment-dropzone--dragover': dragging }"
+          :disabled="field.disabled"
+          @drop.prevent="uploadMedia"
+          @dragover="dragHandler"
+          @dragleave="dragging = false"
+        >
+          <p class="apos-attachment-instructions">
+            <template v-if="dragging">
+              <cloud-upload-icon :size="38"  />
+            </template>
+            <template v-else>
+              <paperclip-icon :size="14" />
```

Comment:
@stuartromanek the `paperclip-icon` was not displayed because of the `v-html` tag, not able to convert Vue code to HTML. So I came up with these 2 messages (and an icon when dragging over the drop zone, but it is just a suggestion, might not be ideal).

Reply:
it looks good!

Reply:
@falkodev please flex `align-items: center` the paperclip and the rest of the label so they are vertically centered. Rotate the paperclip 45deg and put a 5px margin on the right side of the icon